src/components/Calculator.js

```diff
@@ -14,7 +14,7 @@ function Calculator() {
   const { total, next } = state;
 
   return (
-    <CalculatorContainer>
+    <CalculatorContainer className="calculator-container">
       <Display>
```

Comment:
- Hey, great job with the styling, however, let's kindly follow the layout of the design as much as we can. let's kindly align  this section to match the template below:



*yo8ur design*
![Screenshot from 2022-12-07 01-33-30](https://user-images.githubusercontent.com/59397159/206058651-c799f648-710b-4e92-8f22-0333c02658dc.png)

*templeate*
![Screenshot from 2022-12-07 01-33-04](https://user-images.githubusercontent.com/59397159/206058655-9e6575f6-f99a-4d37-afe9-d305886bd50f.png)

Reply:
Hello, @zieeco thank you for your kind words. Side by side isn't as pretty but I'll implement it.

Reply:
This change has been done well